elastic/kibana
# [Response Ops][Actions] Fixing bug where context variables are not passed in when alerts as data doc is available in action params

Resolves https://github.com/elastic/kibana/issues/175624

## Summary

Fixes a bug where context and state variables were not passed to the action params for interpolation if the alert-as-data document is available.

## To Verify
* Create a rule and add an action that fires `For each alert - on check intervals`
* Add a condition to the action, either a KQL query or a time range
* Make sure the action message references some context variables (default ES query or Index Threshold rule messages should include context variables)
* Save the rule and make sure the action fired correctly includes the referenced context variables.
